quiz.rb

```diff
@@ -19,10 +19,14 @@ def is_of_age? age, minimum_age
 # Remember p, print, puts don't return their arguments
 
 def okay_to_drink age, drinking_age
-  # your code goes here
+  is_of_age? age drinking_age
```

Comment:
Your thinking is right here, but you need to wrap the logic in a conditional statement i.e. 

```javascript
if is_of_age? age, drinking_age
   do something
else
   do something
end
```